server/mlcube/migrations/0002_auto_20220624_0853.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.10 on 2022-06-24 08:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mlcube', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='mlcube',
+            old_name='tarball_hash',
+            new_name='additional_files_tarball_hash',
+        ),
+        migrations.RenameField(
+            model_name='mlcube',
+            old_name='tarball_url',
+            new_name='additional_files_tarball_url',
+        ),
+        migrations.AddField(
+            model_name='mlcube',
+            name='image_tarball_hash',
+            field=models.CharField(blank=True, max_length=100),
+        ),
```

Comment:
shouldn't these read "image_hash" and "image_url" instead of "image_tarball..."? I may be confused here.